.github/workflows/send-screenshots-to-pixeleagle.yml

```diff
@@ -0,0 +1,95 @@
+name: Send Screenshots to Pixel Eagle
+
+on:
+  workflow_call:
+    inputs:
+      artifact:
+        required: true
+        type: string
+      commit:
+        required: true
+        type: string
+      branch:
+        required: true
+        type: string
+      os:
+        required: true
+        type: string
+
+jobs:
+  send-to-pixel-eagle:
+    name: Send screenshots to Pixel Eagle
+    # runs on macos because it has the updated curl that will accept the --json flag, and the script is probably not portable to windows
```

Comment:
This can be changed back to `ubuntu-latest` once [Ubuntu 24 becomes the new default](https://github.blog/changelog/2024-05-14-github-hosted-runners-public-beta-of-ubuntu-24-04-is-now-available/). ([Installed software](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#installed-apt-packages))

I won't block on it, though, because who knows when that will be ready. :)